pkgs/by-name/te/terragrunt/package.nix

```diff
@@ -1,18 +1,18 @@
 { lib
-, buildGoModule
+, buildGo123Module
 , fetchFromGitHub
 , go-mockery
 }:
 
-buildGoModule rec {
+buildGo123Module rec {
```

Comment:
This is the default `buildGoModule` currently

Reply:
I changed it, didn't know that it was the default now 😄